src/render-delaying-services.js

```diff
@@ -75,6 +109,21 @@ export function hasRenderDelayingServices(win) {
   return includedServices(win).length > 0;
 }
 
+/**
+ * Function to determine if the passed
+ * Object is a Render Delaying Service
+ * @param {!Object} service
+ * @return {?RenderDelayingService}
+ */
+export function isRenderDelayingService(service) {
```

Comment:
should this just return a boolean?

Reply:
Ah, yes I think you are right, I'll go ahead and make that change 😄